From ff0f9fa012780c74815b159effb60b37f440a7b1 Mon Sep 17 00:00:00 2001
From: Sergey Alyoshin <alyoshin.s@gmail.com>
Date: Thu, 27 Mar 2014 23:26:16 +0400
Subject: [PATCH 1/3] Add .glade file to translation and use xgettext

pygettext is not capable to process .glade file format, which is contain some
translatable messages.
---
 po/Makefile | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/po/Makefile b/po/Makefile
index 54876c6..5dcca6b 100644
--- a/po/Makefile
+++ b/po/Makefile
@@ -18,13 +18,14 @@ srcdir = .
 INSTALL = /usr/bin/install -c
 
 MSGFMT = /usr/bin/msgfmt --statistics
-XGETTEXT = /usr/bin/pygettext
+XGETTEXT = /usr/bin/xgettext
 MSGMERGE = msgmerge
 
 POFILES =  $(wildcard *.po)
 MOFILES =  $(patsubst %.po,%.mo,$(POFILES))
 
-SRCFILES = ../apt-listchanges.py $(wildcard ../apt-listchanges/*.py)
+SRCFILES = ../apt-listchanges.py $(wildcard ../apt-listchanges/*.py) \
+	../apt-listchanges/apt-listchanges.glade
 
 all: $(PACKAGE).pot $(MOFILES)
 
@@ -35,8 +36,9 @@ all: $(PACKAGE).pot $(MOFILES)
 
 $(PACKAGE).pot: $(SRCFILES)
 	if test -f $(XGETTEXT); then \
-	    $(XGETTEXT) --default-domain=$(PACKAGE) --keyword=_ $(SRCFILES); \
+	    $(XGETTEXT) --default-domain=$(PACKAGE) -k_ $(SRCFILES) -o$@; \
 	else \
+	    echo "Warning: $@ is not updated"; \
 	    touch $@; \
 	fi
 
-- 
1.9.1

